contents/teams/developer-experience/objectives.md

```diff
@@ -0,0 +1,12 @@
+## Q1 2026 Objectives
+
+- Enable the creation of **modern, fully isolated products** with CI tests completing in approximately **5 minutes**
+  - Establish Bazel and a scalable, well-defined product structure
+- Clarify and standardize **type usage**, including clear boundaries and expectations for product isolation
```

Comment:
is this python specific?

(is it worth calling out where we're focussing on a particular language and where not)

Reply:
This would cover all languages TS/OpenAPI/Python since it's also about interoperability. Basically the continuation of work in https://github.com/PostHog/posthog/pull/42229